index.html

```diff
@@ -1,362 +1,657 @@
-                HTML
+              <div class="works__card__topics flex">
+                <p
+                  class="works__card__topics__text fw-500 fz-12 lh-16 clr-p-400 bg-p-50"
+                >
+                  html
+                </p>
+                <p
+                  class="works__card__topics__text fw-500 fz-12 lh-16 clr-p-400 bg-p-50"
+                >
+                  css
+                </p>
+                <p
+                  class="works__card__topics__text fw-500 fz-12 lh-16 clr-p-400 bg-p-50"
+                >
+                  javascript
+                </p>
+              </div>
+              <div class="works__card__btn">
+                <button
+                  type="button"
+                  class="btn fw-500 fz-17 lh-24 clr-p-500 bg-white first-btn"
+                  id="pop-button"
+                  id="0"
+                  onclick="Display()"
+                >
+                  See Project
+                </button>
+                <!----------------------------------------- POPUP STRUCTURE --------------------------------------------------------------------- -->
+                <div class="popup-container block-popup">
+                  <div id="popup-modal" class="pop-content">
+                    <div class="headline-popup">
+                      <h1>Tonic</h1>
+                      <a href="#" role="button">
+                        <img
+                          src="images/closing-button.svg"
+                          alt="closing button-popup"
+                          onclick="closePopup()"
+                      /></a>
+                    </div>
+                  </div>
+                  <ul class="small-tags">
+                    <li class="canopy-sm-tag">CANOPY</li>
+                    <li>
+                      <img
+                        src="images/Counter.svg"
+                        alt="counter tags"
+                        role="button"
+                        class="counter"
+                      />
+                    </li>
+                    <li class="second-sm-tag">Back End Dev</li>
+                    <li>
+                      <img
+                        src="images/Counter.svg"
+                        alt="counter tags"
+                        role="button"
+                        class="counter"
+                      />
+                    </li>
+                    <li class="second-sm-tag">2015</li>
+                  </ul>
+                  <div class="popup-image-container">
+                    <img src="images/Snapshoot Portfolio.png" alt="pop-image" />
+                  </div>
+                  <div class="popup-mobile-container">
+                   
+                  </div>
+                  <div class="buttom-content-container">
+                    <div class="paragraph-description">
+                      <p class="paragraph-description-desktop">
```

Comment:
- [ OPTIONAL ] From the project requirement, you should display all these elements dynamically using javascript :relaxed:  I think all this project information should be stored in an array of objects and iterate over displaying them dynamically.